test/unit_tests/braket/aws/test_aws_quantum_task.py

```diff
@@ -480,14 +661,16 @@ def test_from_annealing(device_parameters, aws_session, arn, problem):
     assert task == AwsQuantumTask(
         mocked_task_arn, aws_session, AnnealingQuantumTaskResult.from_string
     )
-
+    annealing_parameters = _create_annealing_device_params(device_parameters, device_arn=arn)
+    # validate(json.loads(device_parameters.json()), annealing_parameters.schema())
+    validate(json.loads(annealing_parameters.json()), annealing_parameters.schema())
```

Comment:
The validation should occur against the schema class, not against the annealing parameters. E.g. `Dwave2000QDeviceParameters.schema()`. It is probably easiest to add this as another variable passed in with each parametrization. 

Also, to mirror how the parameters are passed to the api, the json call should have the keyword `exclude_none=True`.

Reply:
Added in the exclude. The schema change was resulting in failures